internal/pkg/service/common/errors/statuscode.go

```diff
@@ -1,28 +1,44 @@
 package errors
 
 import (
+	"context"
 	"net/http"
 
 	goa "goa.design/goa/v3/pkg"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
+const (
+	StatusClientClosedRequest = 499
```

Comment:
Potrebujeme, aby ked klient ukonci spojenie, tak by to nebola chyba `500 - context canceled`.
Nie je to chyba servera.

Reply:
Neoficialne sa na tuto situaciu pouziva HTTP kod `499`: https://httpstatus.in/499
Jedna sa iba o logs/telemetriu, kedze ked je spojenie ukoncene, tak ziadna odpoved uz klientovi nepride.